operator-common/src/test/java/io/strimzi/operator/common/operator/resource/DeploymentOperatorTest.java

```diff
@@ -46,6 +46,11 @@ protected void mocker(KubernetesClient mockClient, MixedOperation op) {
 
     @Override
     protected DeploymentOperator createResourceOperations(Vertx vertx, KubernetesClient mockClient) {
-        return new DeploymentOperator(vertx, mockClient);
+        return new DeploymentOperator(vertx, mockClient) {
+            @Override
+            protected void setGeneration(Deployment desired, int nextGeneration) {
```

Comment:
Why do we have to override this?

Reply:
It is just for the test reasons. Otherwise I am getting NPE because _something_ is not mocked properly.